src/app/shared/entity.service.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from 'ng-login';
```

Comment:
what is "ng-login"? are you importing a new module or is this a typo and should be ngx-login-client?